settings/default/map.lua

```diff
@@ -55,6 +55,8 @@ xi.settings.map =
     EXP_RATE                = 1.0,
     EXP_LOSS_RATE           = 1.0,
     EXP_PARTY_GAP_PENALTIES = true,
+    -- A party member's experience points are nullified if the level difference with the highest-level party member exceeds this value.
+    EXP_PARTY_GAP_NO_EXP    = 0,
```

Comment:
I'd add another linebreak before the comment and make the comment explain in more detail what this does, what the accepted number range is, and a worked example of what it does. If its away from the other exp settings then the = won't need to be aligned with the rest